README.md

```diff
@@ -50,6 +50,16 @@ git clone https://github.com/uclaacm/YOUR_PROJECT_URL_HERE.git
 cd YOUR_PROJECT_URL_HERE
 ```
 
+The instructions to install Node.js will be different based on which platform you're running. It's heavily advised to install your Node.js using NVM (Node Version Manager) because it's easy to manage a standardized version and update it as needed.
+
+### macOS or Linux
+
+Instructions for installing NVM on macOS and Linux (including WSL) are [here](https://github.com/nvm-sh/nvm#installing-and-updating). At this point you can run `nvm install`. Assuming you've already `cd`ed into the correct directory as mentioned earlier, this will download the LTS (Long-Term Support) version of Node.js for you. Then, run `nvm use` to make sure you've switched to the right version; if it tells you `Now using Node v16` or something similar, you're good to go! 
```

Comment:
Can we specify `lts`? 

Also, feel free to not, but this blob of text is a bit hard to read and can be broken into easier to read chunks like other parts of the `README.md`. I don't think this is too important, so it might not be worth your time

Reply:
Ok sounds good, cool didn’t know that